tasks/GPSDTask.cpp

```diff
@@ -1,4 +1,14 @@
-#include "GPSDTask.hpp"
+
+// workaround for name clash in 
+// usr/include/mach/policy.h and gpsd/3.15/include/gps.h
```

Comment:
Ambiguous statement on my side...

"Shouldn't the workaround be in GPSDTask.hpp ?"